tests/capabilities.js

```diff
@@ -67,19 +46,9 @@ test('capabilities device_formfactor', function(done, fail) {
             capabilities.firefoxAndroid = false;
             capabilities.widescreen = function() { return true; };
             eq_(capabilities.device_formfactor(), '');
-            done();
-        },
-        fail
-    );
-});
+        });
 
-test('capabilities device_type', function(done, fail) {
-    mock(
-        'capabilities',
-        {
-            settings: {}
-        },
-        function (capabilities) {
+        it('capabilities device_type', function() {
```

Comment:
If we're using this style it should read as a sentence e.g. `it('will determine that blah works')` (I know this is WiP so if you've not got to this yet ignore this comment)